Answers.md

```diff
@@ -1,9 +1,22 @@
+Answers
+
 1. If you had to describe semantic HTML to the next cohort of students, what would you say?
 
-2. Describe some differences between ```display: block;``` and ```display: inline;```.
+Semantic HTML is the use of specific html tags, that bring greater meaning and clarity to a website and its content. Effectively allowing for an easier understanding by machines to determine the content of your page, and relevance in a google search or of another similar search engine.
+
+2. Describe some differences between ```display: block;``` and ```display: inline;`.
+
+A block element always starts on a new line, and will take up a block of space from the left of the page to the right. Inline elements do not start a new line and can appear inside of block elements. The difference of which can be esily visualized by placing a border around the different tags.
+
 
 3. What are the 4 areas of the box model?
 
+Border, Margin, Padding, Width/Height
+
 4. While using flexbox, what axis are you using when you use the property: ```align-items: center```?
 
-5. What is the git command to commit staged changes as well as write a message? 
\ No newline at end of file
+Cross-Axis
+
+5. What is the git command to commit staged changes as well as write a message? 
+
+Git is valuable to a team of developers becuase it allows for the team to easily collaborate on a large project that may include countless lines of code. In particular git shines when several developers contribute to the same code, and there are multiple commits which at times cause conflicts that git is typically able to resolve.
```

Comment:
Rod, the answer to this question does not line up with #5. It's asking what the git command is to commit staged changes as well as to write a message. You answered why Git is vauable asset for developers to use. 

EDIT: Question lines up with README but not with your Answers.md. Be careful to proofread your documents before turning them in.